core/domain/exp_domain.py

```diff
@@ -1396,6 +1397,55 @@ def get_translatable_text(self, language_code):
 
         return state_names_to_content_id_mapping
 
+    def get_completed_translations(self, language_code):
+        """Returns all the contents along with the corresponding
+        translations for which translation in the given language has
+        been completed.
+
+        Args:
+            language_code: str. The language code in which translation is
+                required.
+
+        Returns:
+            list(CompletedTranslation). A list of CompletedTranslation object.
+            The object contians
+            content: str. Translation content string.
+            translation: str. Translated Text string.
+        """
+        completed_translations_and_content = []
+        for state_name, state in self.states.items():
+            translations_and_ids = state.get_translated_text_and_ids(
+                language_code)
+            # First appending all the 'content' type translations to the
```

Comment:
See comment in https://docs.google.com/document/d/1fXJ60t3ktfg_wF2SumJLLP7eenMyj2zUDBmFYZXgTkI/edit?disco=AAAAIVvNHfw. This doesn't seem like what we decided upon.